src/WebJobs.Extensions.DurableTask.Analyzers/Analyzers/Orchestrator/MethodAnalyzer.cs

```diff
@@ -0,0 +1,105 @@
+﻿using Microsoft.CodeAnalysis;
```

Comment:
For all the new files in this PR, can you add the copyright header to the tops of the files?